Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,44 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.parent{
+    height: 400px;
+    width: 400px;
+    background-color: #675BC6;
+}
+
+.child-1{
+    height: 20%;
+    width: 20%;
+    top:0;
+    background-color: #60B177;
+}

```

Comment:
Consider adding 'position: relative;' to .child-1, .child-2, .child-3, .child-4, and .child-5 to enable positioning within the parent container.